framework/doc/content/source/meshgenerators/ExtraNodesetGenerator.md

```diff
@@ -2,6 +2,10 @@
 
 !syntax description /Mesh/ExtraNodesetGenerator
 
+!alert note
+The node ids in MOOSE are numbered starting at 0. In Paraview, they are numbered starting
+from 1. Please be mindful of this offset if obtaining the node ids for the [!param](/Mesh/ExtraNodesetGenerator/nodes) parameter from Paraview (and subtract 1).
```

Comment:
```suggestion
!alert note
If you look up node IDs in Paraview for the [!param](/Mesh/ExtraNodesetGenerator/nodes) parameter,
note that you need to subtract 1 from the IDs listed in Paraview, since Paraview starts its indexing
from 1 instead of from 0 like in MOOSE.
```